Project 1/styles.css

```
@@ -0,0 +1,56 @@
+body {
+
+    padding-left: 120px;
+    padding-right: 120px;
+    max-width: 700px;
+    min-width: 500px;
+    background-color: rgb(113, 17, 17);
+    margin: 0 auto;
+}
+
+header {
+    background-color: #f7f7f7;
+    max-width: 300px;
+    min-width: 250px;
+    padding: 30px;
+    margin: 30px auto;
+    border: 5px solid rgb(255, 189, 67);
+}
+
+h1,
+h2 {
+    font-family: 'Spectral', serif;
+    text-align: center;
+}
+
+h1 {
+    font-size: 50px;
+    font-weight: 500px;
```

`font-weight` does not use units -- it is either a keyword (like `bold`) or a number 100-900 without any units, like so:
```suggestion
    font-weight: 500;
```